### PR TITLE
Update ReadMe.md

### PR DESCRIPTION
**Pull request by KitB**

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="https://app.review.ninja/KitB/ReviewNinja-Welcome/pull/1" target="_blank">ReviewNinja

File: ReadMe.md [hunk 1]
```diff
@@ -1 +1,3 @@
-# ReviewNinja-Welcome
\ No newline at end of file
+# ReviewNinja-Welcome
+
+butts!
```
Comment (KitB) [inline ReadMe.md]: sttub

Comment (KitB) [inline ReadMe.md]: rear end though
!fix

Comment (KitB) [inline ReadMe.md]: !fixed

**Pull request merged**